pkg/scalers/couchdb_scaler.go

```diff
@@ -0,0 +1,247 @@
+package scalers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	couchdb "github.com/go-kivik/couchdb/v3"
+	"github.com/go-kivik/kivik/v3"
+	"github.com/go-logr/logr"
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
+	v2 "k8s.io/api/autoscaling/v2"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+	"strconv"
```

Comment:
We split the import section in 3 blocks:
- golang
- deps
- keda
The static analyzer checks this indeed, that's one of the failures in static checks